recipes/diffoscope/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "diffoscope" %}
+{% set version = "84" %}
+{% set sha256 = "7a8e6a2bd3562fa650cad90f4c9a3ec9b070d5a2e55d437f5f28a8a07aecd53f" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
Why do you have both a `script` entry and a `build.sh`? You could delete `build.sh` and do

```yaml
  script: PYTHONHASHSEED=0 python setup.py install --single-version-externally-managed --record record.txt
```